pipelines/SpeciesTreeFromBusco/scripts/collate_busco_results.py

```diff
@@ -17,7 +17,7 @@
 """
 Script for collating per-genome BUSCO cDNA results.
```

Comment:
Include example to run here with parameters - this is where I meant to put it. I think I neglected this on your other script as well, but we should have examples on how to run with parameters in these docstrings.
```suggestion
Script for collating per-genome BUSCO cDNA results.

Example::

    $ python collate_busco_results.py --input input.fofn --genes busco_genes.txt \
        --output per_busco_genes --stats output_stats.tsv --taxa taxa.tsv

```